### PR TITLE
Add Clan Fortune Consults task

### PR DESCRIPTION
**Pull request by MrFizzyBubbs**
Addresses #218

### CODE REVIEW

File: packages/garbo/src/tasks/daily.ts [hunk 1]
```diff
@@ -611,6 +614,25 @@ const DailyTasks: GarboTask[] = [
...
+      have($item`Clan VIP Lounge key`) &&
+      getClanLounge()["Clan Carnival Game"] !== undefined &&
+      isOnline("CheeseFax") &&
+      Clan.getWhitelisted().find(
+        (c) => c.name === "Bonus Adventures from Hell",
+      ) !== undefined,
+    completed: () => get("_clanFortuneConsultUses") >= 3,
+    do: (): void => {
+      Clan.with("Bonus Adventures from Hell", () =>
+        cliExecute("fortune 3038166"),
```
Comment (horrible-little-slime) [inline packages/garbo/src/tasks/daily.ts:629]: either do like ```
cliExecute(`fortune ${getPlayerId("cheesefax")}`)
```
or just add a comment saying it's cheesefax.

Also, isn't cheesefax dead?

Comment (MrFizzyBubbs) [reply packages/garbo/src/tasks/daily.ts:629]: I thought its death was only temporary but who knows.

**Review by horrible-little-slime — APPROVED** [review]
LGTM

**Comment by atbell1** [comment]
This uses Bonus Adventures from Hell if whitelisted, should it not use the pref for VIP clan instead of this hardcoded value? Will cause issues if whitelisted to BAFH but unable to leave current clan as leader, even if current clan is VIP preference

**Comment by Malibu-Stacey** [comment]
> This uses Bonus Adventures from Hell if whitelisted, should it not use the pref for VIP clan instead of this hardcoded value? Will cause issues if whitelisted to BAFH but unable to leave current clan as leader, even if current clan is VIP preference

You can open an issue if you have a problem with the script or propose your own PR if you have a better solution.
This is not appropriate use of comments on a PR that was merged 4 months ago.
Consider this a friendly warning as a first offence. Future warnings will be less friendly (and less warning like).

**Pull request merged**